docs/user/rss/rss.md

```diff
@@ -0,0 +1,115 @@
+(rss)=
+
+# Random structure searching (RSS) workflow
+
+The random structure searching (RSS) approach was initially proposed for predicting crystal structures by generating randomised, sensible structures and optimising them via first-principles calculations ([Phys. Rev. Lett. 97, 045504 (2006)](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.97.045504) and [J. Phys.: Condens. Matter 23, 053201 (2011)](https://iopscience.iop.org/article/10.1088/0953-8984/23/5/053201)). Recently, RSS was expanded into a methodology for exploring and learning potential-energy surfaces from scratch ([Phys. Rev. Lett. 120, 156001 (2018)](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.156001) and [npj Comput. Mater. 5, 99 (2019)](https://www.nature.com/articles/s41524-019-0236-6)). Enhanced with physics-inspired sampling methods, such as Boltzmann-probability biased histograms and CUR, this approach ensures both the significance (low-energy) and diversity of the structures being searched.
+
+## Quick start workflow for immediate use
+
+In `autoplex`, you can use the `RssMaker` module to build a complete ML-RSS potential model from scratch. `RssMaker` accepts customizable parameters either through a YAML configuration file or by directly specifying arguments in the `make` method. These [parameters](#rssmaker-parameters) control various aspects of the RSS workflow, including randomized structure generation, sampling methods, DFT labeling, data preprocessing (e.g., regularization), potential fitting hyperparameters, and iterative settings for RSS. The minimal input required is simply specifying the chemical formula of the target system by the argument `tag`.
```

Comment:
What us really unclear to m is if the user needs AIRSS/buildcell **and** lammps for the RSS-wf or if it depends on what the user wants to do

Reply:
It depends on buildcell only if you want to use it with GAP. Only if you wamt to go to mace, lammps is required.

Reply:
Then I guess we can add this information here :)

Reply:
Maybe in a note on required installations. I wouldn't put this into this text

Reply:
I have added this hint to the LAMMPS installation.